github/event_types.go

```diff
@@ -146,6 +146,18 @@ type EditChange struct {
 	} `json:"body,omitempty"`
 }
 
+// IntegrationEvent is triggered when an integration is created or modified.
+// The Webhook event name is "integration_installation" or
+// "integration_installation_repositories".
+//
+// GitHub docs: https://developer.github.com/early-access/integrations/webhooks/
+type IntegrationEvent struct {
+	Action            *string       `json:"action,omitempty"`
```

Comment:
Can you please document this field?  For example:
```go
// The action that was performed. Possible values for an "integration_installation"
// event are: "created", "deleted". Possible values for an "integration_installation_repositories"
// event are: "added", "removed".
Action ...
```
(Following the style of the documentation found in other files in this repo.)